lang/LangSource/SC_CLIOptions.hpp

```diff
@@ -0,0 +1,31 @@
+#include "boost/program_options.hpp"
+#include <string>
+#include "SC_TerminalClient.h"
+
+namespace po = boost::program_options;
+using namespace std;
+
+namespace SC_CLI {
+class CLIOptions {
+public:
+    CLIOptions() {};
```

Comment:
Please no semi-colon after inline function definitions :) Also, this constructor does nothing, so you can just omit it altogether.